src/peerd/runtime.rs

```diff
@@ -327,7 +333,7 @@ impl esb::Handler<ServiceBus> for Runtime {
             // RPC command bus, only accept BusMsg::Info
             (ServiceBus::Info, BusMsg::Info(req)) => self.handle_info(endpoints, source, req),
             // Internal peerd bridge for inner communication, accept all type of request
```

Comment:
Fix comment, not all type of req anymore

Reply:
Done